climakitae/util/generate_gwl_tables.py

```diff
@@ -100,7 +104,7 @@ def build_timeseries(variable, model, ens_mem, scenarios):
                 if not df_scenario.empty:
                     with xr.open_zarr(
                         fs.get_mapper(df_scenario.zstore.values[0]), decode_times=False
-                    ) as temp:
+                    ) as temp:  # BUG: Some scenarios not returning a full predictive period of values (i.e. not returning time period of 2015-2100 of data)
```

Comment:
Does the comment refer to the “Future Model Data” in the data selection UI? I’m a bit confused and think that part of the _`generate_gwl_tables.py` Notes_ may be helpful to the debugger(s) and worth including here, or another place that's accessible.

Reply:
This refers to the fact that some of the models have limited amounts of data in their zarr formats (slicing for 300 timestamps vs 1000). This is beyond the scope of we can do on our end, as we can't un-slice the original data. I expanded on this note further in this doc: https://docs.google.com/document/d/1pzrjvp7c5FUISvwVXLQbFf3H-OsvaKMDSKEKR8WtBNI/edit .

I can also link this doc in our codebase, in case that would make things clearer too.